static/app/components/acl/feature.tsx

```diff
@@ -44,6 +44,7 @@ type Props = {
    * function.
    */
   hookName?: keyof FeatureDisabledHooks;
+  organizationAllowNull?: undefined | true;
```

Comment:
Are we using this type instead of a bool for some sort of backwards compatibility?

Reply:
Yeah, typescript was complaining about using bool. Presumably because this value cannot be false, can only be true or undefined